test/Chainweb/Test/Pact/TxGen.hs

```diff
@@ -0,0 +1,616 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE MultiWayIf #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE QuasiQuotes #-}
+{-# OPTIONS_GHC -fno-warn-unused-matches #-}
+{-# OPTIONS_GHC -fno-warn-type-defaults #-}
+-- |
+-- Module: Chainweb.Test.TxGen
```

Comment:
I don't see this module listed in the `.cabal` file.  Also, this comment doesn't match the module name.

Reply:
I must have left it in by accident. It is a module that I was going to create but later realized was more trouble than it was worth. I'll delete the line.